test/deprecated.jl

```diff
@@ -539,6 +539,108 @@ end
     @test_throws ArgumentError join(df1, df2, df3, on=:id, kind=:xxx)
 end
 
+@testset "eachcol(df, true)" begin
+    df = DataFrame(a=1:3, b=4:6, c=7:9)
+    @test eachcol(df)[1] === last(eachcol(df, true)[1])
+    @test eachcol(df)[1] === last(eachcol(df, true)[1])
+
+
+    # show for eachcol(df, true) is breaking but this is a minor thing
```

Comment:
Just remove this?

Reply:
ok - removed the whole portion of commented-out tests.